src/ViewHelpers/PaginationViewHelper.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace Portal\ViewHelpers;
+
+class PaginationViewHelper
+{
+
+    /**
+     * Takes in count of pages required and current page, and returns pagination for applicants list.
+     *
+     * @param $page
+     * @param $count
+     *
+     * @return string
+     */
+    public static function applicantsPagination($page, $count)
+    {
+        $next = (strval($page+1));
+        $prev = (strval($page-1));
+
+        $applicantPagination = '';
+
+        $applicantPagination .= '<nav aria-label="Page navigation mt-5">';
+        $applicantPagination .= '<ul class="pagination justify-content-center">';
+        $applicantPagination .= '<li class="page-item ';
+        if ($page <= 1) {
+            $applicantPagination .= 'disabled';
```

Comment:
Can you add some CSS for this disabled class that sets pointer-events: none; on the a tag? This way the user wont even be able to click on it.